ima-policy/ima-policy

```diff
@@ -0,0 +1,1046 @@
+# PROC_SUPER_MAGIC
+dont_measure fsmagic=0x9fa0
+# SYSFS_MAGIC
+dont_measure fsmagic=0x62656572
+# DEBUGFS_MAGIC
+dont_measure fsmagic=0x64626720
+# TMPFS_MAGIC
+dont_measure fsmagic=0x1021994
+# RAMFS_MAGIC (missing)
+# DEVPTS_SUPER_MAGIC
+dont_measure fsmagic=0x1cd1
+# BINFMTFS_MAGIC
+dont_measure fsmagic=0x42494e4d
+# SECURITYFS_MAGIC
+dont_measure fsmagic=0x73636673
+# SELINUX_MAGIC
+dont_measure fsmagic=0xf97cff8c
+# SMACK_MAGIC (new)
+dont_measure fsmagic=0x43415d53
+# CGROUP_SUPER_MAGIC
+dont_measure fsmagic=0x27e0eb
+# CGROUP2_SUPER_MAGIC (new)
+dont_measure fsmagic=0x63677270
+# NSFS_MAGIC
+dont_measure fsmagic=0x6e736673
+# EFIVARFS_MAGIC (new)
+dont_measure fsmagic=0xde5e81e4
+
+# base_file_type, base_ro_file_type
+dont_measure obj_type=etc_runtime_t
+dont_measure obj_type=system_conf_t
+dont_measure obj_type=system_db_t
+dont_measure obj_type=tmp_t
+dont_measure obj_type=var_lib_t
+dont_measure obj_type=var_lock_t
+dont_measure obj_type=var_run_t
+dont_measure obj_type=var_spool_t
+
+# for i in $(seinfo -alockfile -x | grep _t | tr '[:space:]' " "); do echo "dont_measure obj_type=$i"; done
```

Comment:
nit: it would be a little more robuster to limit the grep pattern as `_t$`? also it could be simply written with a single sed invocation: `seinfo -alockfile -x | sed -n 's/^\t\(.*_t\)$/dont_measure obj_type=\1/p'`.